hierarqcal/utils.py

```diff
@@ -14,6 +14,7 @@
 import numpy as np
 import matplotlib.pyplot as plt
```

Comment:
Remember to format the file with black

Reply:
The default is already black, if i understand it correctly. Can you let you know exactly what do you to fix for the matplotlib style?

Reply:
Sorry, I meant [black codeformatter](https://black.readthedocs.io/en/stable/), nothing to do with matplotlib, I just wanted to comment on the top of the file xD. 

Should be as simple as pip install black -> format document in vs code

Reply:
Done. Added changes after that.